### ISSUE TITLE
perlfilter.pod (and others) not mentioned in perl.pod

### ISSUE THREAD

**Issue by p5pRT** [Perl/perl5#1250]
Migrated from [rt.perl.org#2244](https://rt-archive.perl.org/perl5/Ticket/Display.html?id=2244) (status was 'resolved')

Searchable as RT2244$

**Comment by p5pRT** [comment]
### From dcd@tc.fluke.com

#### Created by dcd@tc.fluke.com



I noticed that perlfilter\.pod is not mentioned in perl\.pod\.

&nbsp;    perlfilter is mentioned in perldelta\.pod and perltoc\.pod but
&nbsp;    that doesn't really count in my book :\-\)

The following other pod files are also not mentioned in perl\.pod

&nbsp;    perllexwarn\.pod
&nbsp;    perlunicode\.pod

&nbsp;    Win32\.pod
&nbsp;    perlamiga\.pod
&nbsp;    perlcygwin\.pod
&nbsp;    perldos\.pod
&nbsp;    perlhpux\.pod
&nbsp;    perlos2\.pod
&nbsp;    perlos390\.pod
&nbsp;    perlvms\.pod
&nbsp;    perlwin32\.pod

The individual faq documents aren't mention explicitly\, but
are referenced from perlfaq\.pod

&nbsp;    perlfaq1\.pod
&nbsp;    perlfaq2\.pod
&nbsp;    perlfaq3\.pod
&nbsp;    perlfaq4\.pod
&nbsp;    perlfaq5\.pod
&nbsp;    perlfaq6\.pod
&nbsp;    perlfaq7\.pod
&nbsp;    perlfaq8\.pod
&nbsp;    perlfaq9\.pod

<details><summary>Perl Info</summary>

```


Site configuration information for perl v5.5.670:

Configured by dcd at Wed Mar  1 13:09:07 PST 2000.

Summary of my perl5 (revision 5.0 version 5 subversion 670) configuration:
  Platform:
    osname=linux, osvers=2.2.15pre10, archname=i686-linux
    uname='linux dd 2.2.15pre10 #2 thu feb 24 09:36:58 pst 2000 i686 '
    config_args='-Doptimize=-g -de -Dcf_email=dcd@tc.fluke.com'
    hint=previous, useposix=true, d_sigaction=define
    usethreads=undef use5005threads=undef useithreads=undef usemultiplicity=undef
    useperlio=undef d_sfio=undef uselargefiles=define 
    use64bitint=undef use64bitall=undef uselongdouble=undef usesocks=undef
  Compiler:
    cc='cc', optimize='-g', gccversion=2.7.2.3
    cppflags='-DDEBUGGING -I/usr/local/include'
    ccflags ='-DDEBUGGING -I/usr/local/include'
    stdchar='char', d_stdstdio=define, usevfork=false
    intsize=4, longsize=4, ptrsize=4, doublesize=8
    d_longlong=define, longlongsize=8, d_longdbl=define, longdblsize=12
    ivtype='long', ivsize=4, nvtype='double', nvsize=8, Off_t='off_t', lseeksize=4
    alignbytes=4, usemymalloc=n, prototype=define
  Linker and Libraries:
    ld='cc', ldflags =' -L/usr/local/lib'
    libpth=/usr/local/lib /lib /usr/lib
    libs=-lgdbm -ldbm -ldb -ldl -lm -lc
    libc=/lib/libc.so.5.4.44, so=so, useshrplib=false, libperl=libperl.a
  Dynamic Linking:
    dlsrc=dl_dlopen.xs, dlext=so, d_dlsymun=undef, ccdlflags='-rdynamic'
    cccdlflags='-fpic', lddlflags='-shared -L/usr/local/lib'

Locally applied patches:
    


@INC for perl v5.5.670:
    /usr/local/lib/perl5/5.5.670/i686-linux
    /usr/local/lib/perl5/5.5.670
    /usr/local/lib/perl5/site_perl/5.5.670/i686-linux
    /usr/local/lib/perl5/site_perl/5.5.670
    /usr/local/lib/perl5/site_perl
    .


Environment for perl v5.5.670:
    HOME=/home/dcd
    LANG (unset)
    LANGUAGE (unset)
    LD_LIBRARY_PATH (unset)
    LOGDIR (unset)
    PATH=/home/dcd/bin:/sbin:/usr/local/bin:/bin:/usr/bin:/usr/X11/bin:/usr/games:/usr/local/samba:/home/hobbes/tools/scripts:/home/hobbes/tools/linux:/usr0/hobbes/tools/scripts:/usr0/dcd/bin:/apps/general/bin:/usr/public
    PERL_BADLANG (unset)
    SHELL=/bin/bash


```
</details>

**Comment by p5pRT** [comment]
### From The RT System itself

closed&#8203;: Looks ok in @&#8203;7385\.
